### PR TITLE
ddl: Update the max interval of checking job

### PR DESCRIPTION
**Pull request by zimulala**
3s is a bit long.

### CODE REVIEW

**Comment by hanfei1991** [comment]
LGTM

File: ddl/ddl.go [hunk 1]
```diff
@@ -406,6 +406,15 @@ func (d *ddl) OwnerManager() OwnerManager {
 	return d.ownerManager
 }
 
+func checkJobMaxInterval(job *model.Job) time.Duration {
```
Comment (shenli) [inline ddl/ddl.go]: Change the log level to debug in line 462.
Rest LGTM

**Comment by zimulala** [comment]
PTAL @shenli

**Comment by shenli** [comment]
LGTM

**Pull request merged**